homeassistant/components/prometheus/__init__.py

```diff
@@ -136,12 +136,12 @@ def __init__(
         self._override_metric = override_metric
         self._default_metric = default_metric
         self._filter = entity_filter
-        self._sensor_metric_handlers = [
-            self._sensor_override_component_metric,
-            self._sensor_override_metric,
-            self._sensor_attribute_metric,
-            self._sensor_default_metric,
-            self._sensor_fallback_metric,
+        self._metric_name_handlers = [
+            self._metric_name_override_component_metric,
+            self._metric_name_override_metric,
+            self._metric_name_attribute_metric,
+            self._metric_name_default_metric,
+            self._metric_name_fallback_metric,
```

Comment:
Perhaps `_metric` suffix is a bit unnecessary now.

Reply:
Done!